posthog/api/test/__snapshots__/test_dashboard.ambr

```diff
@@ -0,0 +1,430 @@
+# name: TestDashboard.test_retrieve_dashboard
+  '
+  SELECT "posthog_user"."id",
+         "posthog_user"."password",
+         "posthog_user"."last_login",
+         "posthog_user"."first_name",
+         "posthog_user"."last_name",
+         "posthog_user"."is_staff",
+         "posthog_user"."is_active",
+         "posthog_user"."date_joined",
+         "posthog_user"."uuid",
+         "posthog_user"."current_organization_id",
+         "posthog_user"."current_team_id",
+         "posthog_user"."email",
+         "posthog_user"."temporary_token",
+         "posthog_user"."distinct_id",
+         "posthog_user"."email_opt_in",
+         "posthog_user"."anonymize_data",
+         "posthog_user"."toolbar_mode",
+         "posthog_user"."events_column_config"
+  FROM "posthog_user"
+  WHERE "posthog_user"."id" = 2
+  LIMIT 21
+  '
+---
+# name: TestDashboard.test_retrieve_dashboard.1
+  '
+  SELECT "posthog_team"."id",
+         "posthog_team"."uuid",
+         "posthog_team"."organization_id",
+         "posthog_team"."api_token",
+         "posthog_team"."app_urls",
+         "posthog_team"."name",
+         "posthog_team"."slack_incoming_webhook",
+         "posthog_team"."created_at",
+         "posthog_team"."updated_at",
+         "posthog_team"."anonymize_ips",
+         "posthog_team"."completed_snippet_onboarding",
+         "posthog_team"."ingested_event",
+         "posthog_team"."session_recording_opt_in",
+         "posthog_team"."signup_token",
+         "posthog_team"."is_demo",
+         "posthog_team"."access_control",
+         "posthog_team"."test_account_filters",
+         "posthog_team"."path_cleaning_filters",
+         "posthog_team"."timezone",
+         "posthog_team"."data_attributes",
+         "posthog_team"."primary_dashboard_id",
+         "posthog_team"."correlation_config",
+         "posthog_team"."session_recording_retention_period_days",
+         "posthog_team"."plugins_opt_in",
+         "posthog_team"."opt_out_capture",
+         "posthog_team"."event_names",
+         "posthog_team"."event_names_with_usage",
+         "posthog_team"."event_properties",
+         "posthog_team"."event_properties_with_usage",
+         "posthog_team"."event_properties_numerical"
```

Comment:
@Twixes I'm pretty sure these fields are being requested from the `TeamMemberAccessPermission` class. But I couldn't figure out exactly where. I think this is slowing down requests any where that permission is used, so it's probably worth digging into.

Reply:
These are all deprecated. Perhaps we should also `rm -rf` the data in them?

Reply:
Yeah, this isn't actually used by anything at all.

Reply:
Nice! Let's drop them?

Reply:
I didn't know we had this policy. I would like to challenge it but maybe another time 😅

Reply:
I'd love to delete all that old crap, but we can't really :'(.

I really should start keeping a list, but we've had for sure more than 10 downtimes ([one really recently](https://github.com/PostHog/postmortem/blob/master/2022-02-18-rename-tags-column.md)) when we rename or delete a column. 

We can mitigate it on cloud by slowly running migrations after one another, to 1) stop using the old columns everywhere, including all "select each,column,that,django,knows,about,explicitly,listed from table" serializers used all around django, 2) drop the columns. However this won't work for self-hosted users who might even upgrade from posthog 1.10 to 1.20 in one go.